### PR TITLE
papr:  update for release of F28

### PR DESCRIPTION
**Pull request by miabbott**
Let's start testing the tests against F28 and switch to using the
'updates' stream on Fedora 27 AH.

### CODE REVIEW

**Comment by miabbott** [comment]
@jlebon could you take a peek when you get a chance?

File: .papr.yml [hunk 1]
```diff
@@ -50,8 +69,6 @@ cluster:
 context: centos/7/atomic
 
 tests:
-  # skipping kernel_avc_denied due to https://bugzilla.redhat.com/show_bug.cgi?id=1536991
-  #  - should be fixed in CentOS 7.5
   # skipping 'default_t' checks until atomic 1.22 lands with (projectatomic/atomic#1185)
-  - ./.test_director --skip-tags kernel_avc_denied,default_file_label
```
Comment (jlebon) [inline .papr.yml]: Minor: should this be a separate commit? I don't see anything about it in the commit msg.

Comment (miabbott) [reply .papr.yml]: Sure, that would be the proper thing to do

**Comment by miabbott** [comment]
Split the commits and force pushed :arrow_up:

**Comment by jlebon** [comment]
LGTM!
